src/screens/DashboardScreen.tsx

```diff
@@ -119,6 +162,47 @@ const DashboardScreen = ({ navigation, ...props }: ScreenProps) => {
           </AppText>
         </EmptyPlaceholder>
       )}
+      <Portal>
+        <BottomModal
+          isOpen={isBackupReminderModalOpen}
+          onClose={() => setIsBackupReminderModalOpen(false)}
+          Content={(props) => (
+            <ModalContent verticalGap {...props}>
+              <ScreenSection>
+                <BottomModalScreenTitle>
+                  {isNewWallet ? 'Welcome to your new wallet! 👋' : "Let's verify! 😌"}
```

Comment:
Nit: Maybe keeping the title on one line could be nicer :) For the first one, what about saying something like: Welcome! 👋  or Warm welcome! 👋 ?

Reply:
"A warm welcome 👋"? https://englishgrammarmaster.quora.com/Which-is-correct-a-warm-welcome-or-warm-welcome

BTW, we already say "Welcome to Alephium! Enjoy your new wallet!" in the previous screen. Maybe we could say something simpler in the modal, like:

"Hello there 👋"

WDYT?

Reply:
Yes! I like it.

Reply:
Next PR 😇